client/src/resources/authentication/LogoutButton.js

```diff
@@ -4,20 +4,24 @@ import { connection } from './../headerState.js';
 import { browserHistory } from 'react-router';
 
 class Logout extends React.Component {
-	constructor(props) {
-		super(props);
-		this.logout = this.logout.bind(this);
-	}
+  constructor(props) {
+    super(props);
+    this.logout = this.logout.bind(this);
+  }
 
-	logout() {
-		document.cookie = "token=nil";
-		this.props.changeAuth();
-		browserHistory.push('/');
-	}
+  logout() {
```

Comment:
When the cookie times out, it send the user a message saying they are no longer authorized and re-routes them to the login